corehq/apps/app_manager/static/app_manager/js/language-profiles.js

```diff
@@ -0,0 +1,70 @@
+hqDefine('app_manager/js/language-profiles.js', function () {
+    function Profile(profile_langs, name, id) {
+        var self = this;
+        this.id = id;
+        this.langs = ko.observableArray(profile_langs);
+        this.name = ko.observable(name);
+        this.defaultLang = ko.observable(profile_langs[0]);
+    }
+    function ProfileManager(app_profiles, app_langs) {
+        var self = this;
+        this.app_profiles = ko.observableArray([]);
+        this.app_langs = app_langs;
+        this.saveButton = COMMCAREHQ.SaveButton.init({
+            unsavedMessage: gettext("You have unsaved changes to your build profiles"),
+            save: function() {
+                var postProfiles = [];
+                _.each(self.app_profiles(), function(element, index, list) {
+                    // move default lang to first element of array
+                    var postLangs = element.langs();
+                    postLangs.splice(postLangs.indexOf(element.defaultLang()), 1);
+                    postLangs.unshift(element.defaultLang());
+                    postProfiles.push({
+                        'name': element.name(),
+                        'langs': postLangs,
+                        'id': element.id,
+                    });
+                });
+                self.saveButton.ajax({
+                    url: 'profiles/',
```

Comment:
Not a big deal, but this is a little brittle. I get why you didn't want to do a reverse `{% url ... %}` in the template and pass in the url to the javascript, but maybe just add a comment about which django this view is intending to reference. At least that way, it's greppable if someone is later refactoring and looking for usages of the view.